.eslintrc.json

```diff
@@ -4,10 +4,11 @@
         "es2021": true
     },
     "extends": [
-        "plugin:react/recommended",
         "eslint:recommended",
         "airbnb",
-        "plugin:@typescript-eslint/recommended"
+        "plugin:@typescript-eslint/recommended",
+        "plugin:react/recommended",
```

Comment:
Moved the react plugin later in the extends list. Some of the react specific rules were being ignored/overrode before.